use_case_examples/hybrid_model/compile_hybrid_llm.py

```diff
@@ -0,0 +1,140 @@
+"""Showcase for the hybrid model converter."""
+
+import os
+from copy import deepcopy
+from pathlib import Path
+from typing import List, Union
+
+import torch
+from concrete.fhe import Configuration, ParameterSelectionStrategy
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from concrete.ml.torch.hybrid_model import HybridFHEModel
+
+
+def compile_model(
+    model_name: str,
+    model: torch.nn.Module,
+    inputs: torch.Tensor,
+    module_names: Union[str, List],
+    expected_accuracy,
+    models_dir: Path,
+):
+    """Run the test for any model with its private module names."""
+
+    # Enable multi params/precision
+    configuration = Configuration(
+        single_precision=False,
+        parameter_selection_strategy=ParameterSelectionStrategy.MULTI,
+    )
+
+    # Create a hybrid model
+    hybrid_model = HybridFHEModel(model, module_names)
+    hybrid_model.compile_model(
+        inputs,
+        n_bits=8,
+        # setting it to None is not enough -> weird
+        rounding_threshold_bits=None,
+        configuration=configuration,
+    )
+
+    # Sanity checks
+    logits_simulate = hybrid_model(inputs, fhe="simulate").logits
+    logits_disable = hybrid_model(inputs, fhe="disable").logits
+    logits_original = model(inputs).logits
+    # Ensure logits_disable and logits_original return the same output for the logits
```

Comment:
Could you add a space before the comments?

Reply:
Do you mean a blank line?

Reply:
Yes blank line ^^

Reply:
Added

Reply:
thanks!